internal/db/start/start.go

```diff
@@ -122,7 +124,12 @@ func StartDatabase(ctx context.Context, fsys afero.Fs, w io.Writer, options ...f
 		},
 	}
 	if utils.Config.Db.MajorVersion <= 14 {
-		config.Entrypoint = nil
+		config.Entrypoint = []string{"sh", "-c", `
+			cat <<'EOF' > /docker-entrypoint-initdb.d/supabase_schema.sql
+` + _supabaseSchema + `
+EOF
+			docker-entrypoint.sh postgres -D /etc/postgresql
+		`}
```

Comment:
I had to do that to make the start also work for postgres v14 and v13. I'm not sure it's the best option tough, maybe I'm missing why it was nil before 🤔 

I also wonder if in those cases, we should migrate the part where there is "_realtime" schema seeding inside `13.sql` and `14.sql` to it or not.

Reply:
Yup, this is a good fix. Previously we set entrypoint to nil to use the default value that came with 13 and 14 images.